crates/stdarch-verify/src/lib.rs

```diff
@@ -37,14 +37,33 @@ fn functions(input: TokenStream, dirs: &[&str]) -> TokenStream {
 
     let mut functions = Vec::new();
     for &mut (ref mut file, ref path) in &mut files {
-        for item in file.items.drain(..) {
-            if let syn::Item::Fn(f) = item {
-                functions.push((f, path))
+        for mut item in file.items.drain(..) {
+            match item {
+                syn::Item::Fn(f) => functions.push((f, path)),
+                syn::Item::Mod(ref mut m) => {
```

Comment:
stdsimd-verify did not scan for function items within inline modules (`mod { ... }`) - since tests are inside a `mod tests { ... }` this is required to add them.